apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -601,25 +601,28 @@ connector_config_fields() ->
         {ssl, mk(ref(ssl_client_opts), #{})}
     ].
 
-producer_opts() ->
+producer_opts(ActionOrBridgeV1) ->
     [
         %% Note: there's an implicit convention in `emqx_bridge' that,
         %% for egress bridges with this config, the published messages
         %% will be forwarded to such bridges.
         {local_topic, mk(binary(), #{required => false, desc => ?DESC(mqtt_topic)})},
-        parameters_field(),
+        parameters_field(ActionOrBridgeV1),
         {resource_opts, mk(ref(resource_opts), #{default => #{}, desc => ?DESC(resource_opts)})}
     ].
 
 %% Since e5.3.1, we want to rename the field 'kafka' to 'parameters'
 %% Hoever we need to keep it backward compatible for generated schema json (version 0.1.0)
 %% since schema is data for the 'schemas' API.
-parameters_field() ->
+parameters_field(ActionOrBridgeV1) ->
+    OverriddenV1 = <<"0.1.0">> =:= get(emqx_bridge_schema_version),
```

Comment:
since the new approach passes down the version as argument,
i think we can delete put/get pair.
i.e. 0.1.0 is always v1 ?

Reply:
Apparently, the old behavior suggests that it's supposed to be `v1` when this thing is [set here](https://github.com/emqx/emqx/blob/01895fd660f6a5ac11b26416372bfc47d190fb5c/apps/emqx_conf/src/emqx_conf.erl#L193-L201).

The new testcases don't see `kafka` in the `/actions` API schema, nor `parameters` in `/bridges`.  Checking swagger manually also indicates that.  Not sure if there's somewhere else to check.

Reply:
Actually, the old behavior apparently set `kafka` as the key name for both /actions and /bridges when generating the docs.  :thinking:

Reply:
Removed.  :heavy_check_mark: